source/_components/qld_bushfire.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: page
+title: "Queensland Bushfire Alert"
+description: "Instructions on how to integrate the Queensland Bushfire Alert feed into Home Assistant."
+date: 2019-06-11 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: geo_location.png
+ha_category: Geolocation
+ha_iot_class: Cloud Polling
+ha_release: 0.95
+---
+
+The `qld_bushfire` platform lets you integrate a 
+[GeoRSS feed](https://www.ruralfire.qld.gov.au/map/Pages/default.aspx) 
+containing bushfire alerts for Queensland. It retrieves alerts from a feed 
+and shows information of those alerts filtered by distance to Home Assistant's 
+location.
+
+Entities are generated, updated and removed automatically with each update 
+from the feed. Each entity defines latitude and longitude and will be shown 
+on the default map automatically, or on a map card by defining the source 
+`qld_bushfire`. The distance in kilometers is available as the state 
+of each entity.
+
+<p class='img'>
+  <img src='{{site_root}}/images/screenshots/qld-bushfire-feed-map.png' />
+</p>
+
+The data is updated every 5 minutes.
+
+<p class='note'>
+The material used by this integration is provided under the [Creative Commons Attribution 4.0 license](https://creativecommons.org/licenses/by/4.0/legalcode).
+It has only been modified for the purpose of presenting the material in Home Assistant.
+Please refer to the [creator's copyright notice](https://www.qfes.qld.gov.au/Pages/copyright.aspx) for more information.
+</p>
+
+## {% linkable_title Configuration %}
+
+To integrate the Queensland Bushfire Alert feed, add the following lines to 
+your `configuration.yaml`.
+
+```yaml
+# Example configuration.yaml entry
+geo_location:
+  - platform: qld_bushfire
+```
+
+{% configuration %}
+categories:
+  description: List of category names found in the feed. Only alerts from the feed that match any of these categories are included. Valid categories are 'Emergency Warning', 'Watch and Act', 'Advice', 'Notification' and 'Information'.
+  required: false
+  type: list
+  default: None. Any alert regardless of its category will be included.
+radius:
+  description: The distance in kilometers around Home Assistant's coordinates in which bushfire alerts are included.
+  required: false
+  type: float
+  default: 20.0
+latitude:
+  description: Latitude of the coordinates around which events are considered.
+  required: false
+  type: string
+  default: Latitude defined in your `configuration.yaml`
```

Comment:
This no longer has to be `configuration.yaml`, since it is now manageable in the user interface.

```suggestion
  default: Latitude defined in your configuration.
```

Reply:
OK